warrior/Framework/ClassUtils/WSelenium/browser_mgmt.py

```diff
@@ -538,6 +540,11 @@ def _make_ff(self, webdriver_remote_url, desired_capabilites, profile_dir, **kwa
                 # This is for internal testing needs...some https cert is not secure
                 # And firefox will need to know how to handle it
                 ff_capabilities['acceptInsecureCerts'] = True
+                
+                if not fileExists(str(binary)):
```

Comment:
We need to handle two things here:
1. When user doesn't give any value for binary tag in IDF - Info_msg: No value given for binary, default firefox will be used
2. When the user provides some path but that doesn't exist - Warning_msg: Given binary path doesn't exist, default firefox will be used.

Pt. 2 is already there, just add Pt. 1.

Reply:
Addressed the comments.